packages/url_launcher/url_launcher/README.md

```diff
@@ -8,6 +8,22 @@ iOS, Android, web, Windows, macOS, and Linux.
 ## Usage
 To use this plugin, add `url_launcher` as a [dependency in your pubspec.yaml file](https://flutter.dev/platform-plugins/).
 
+## Installation
+
+### iOS 
+Add any URL scheme passed to canLaunch in your Info.plist file.
+
+Example:  
+```
+<key>LSApplicationQueriesSchemes</key>
+<array>
+  <string>https</string>
+  <string>http</string>
+</array>
+```
+
+See [canOpenURL](https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl) for more details.
```

Comment:
Nit: the standard way to refer to an OjbC method without context includes the class, and the full signature (which includes the `-` and the `:`), like: `-[UIApplication canOpenURL:]` 
```
See [`-[UIApplication canOpenURL:]`](...) for more details.
```